core/service/src/components.rs

```diff
@@ -283,7 +283,7 @@ pub trait ServiceFactory: 'static + Sized {
 	/// ImportQueue for light clients
 	type LightImportQueue: consensus_common::import_queue::ImportQueue<Self::Block> + 'static;
 
-	//TODO: replace these with a constructor trait. that TransactionPool implements. (#1242)
+	//FIXME: replace these with a constructor trait. that TransactionPool implements. (#1242)
```

Comment:
```suggestion
	//TODO: replace these with a constructor trait. that TransactionPool implements. (#1242)
```

switch back to ToDo, not waiting for other code.